_includes/docs-info.html

```diff
@@ -42,17 +42,19 @@ <h6>For Users</h6>
         <a name="tutorials"></a>
         <h5>Tutorials for the finer steps</h5>
             <div class="row docs-details">
-                <a href="https://bit.ly/2Fyn97v">Spherical Harmonic Representation Point Distribution Models (SPHARM-PDM)</a> Last updated July 21st 2017.
+                <a href="https://bit.ly/3q73CTo">Spherical Harmonic Representation Point Distribution Models (SPHARM-PDM)</a> Last updated September 2022.
                 <br/>
-                <a href="https://bit.ly/2WsFiun">Study-specific shape analysis (GROUPS)</a> Last updated July 16th 2018.
+                <a href="https://bit.ly/3q6v1oo">Study-specific shape analysis (GROUPS)</a> Last updated September 2022.
                 <br/>
-                <a href="http://bit.ly/2uVYche">Shape Regression</a> Last updated February 5th 2019.
+                <a href="https://bit.ly/3KKLWGO">Shape Regression</a> Last updated September 2022.
                 <br/>
-                <a href="https://bit.ly/2HYbHVA">Shape Variation Analyzer</a> Last updated December 9th 2019.
+                <a href="https://bit.ly/3cHG704">Shape Variation Analyzer</a> Last updated September 2022.
```

Comment:
https://docs.google.com/presentation/d/1sa_DFisrq-yplJln2uueVKa9VjV2A1N4dDWo91EN09E/present?slide=id.p11
https://docs.google.com/presentation/d/1sa_DFisrq-yplJln2uueVKa9VjV2A1N4dDWo91EN09E/present?slide=id.p12
https://docs.google.com/presentation/d/1sa_DFisrq-yplJln2uueVKa9VjV2A1N4dDWo91EN09E/present?slide=id.p17
https://docs.google.com/presentation/d/1sa_DFisrq-yplJln2uueVKa9VjV2A1N4dDWo91EN09E/present?slide=id.p35

Some of hyperlinks don't appear to work for me. I am unable to click on them. I can click on the ShapeVariationAnalyzer github repo link on the last slide.

Reply:
These seem to be working for me.

Reply:
Interesting. I updated the links such that I can click on them as well.